testing/termite/APKBUILD

```diff
@@ -0,0 +1,66 @@
+# Contributor: Ian Bashford <ianbashford@gmail.com>
+# Maintainer: Ian Bashford <ianbashford@gmail.com>
+pkgname=termite
+pkgver=15
+pkgrel=0
+_vtever=0.50.2-ng
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="x86_64"
```

Comment:
Can't this be done for more arches ?

Reply:
I've changed it to "all" - I don't see why it shouldn't work, as its a simple app.
The travis CI build only looks like it runs for x86_64 - if there's any other way I can check they all work I'd be happy to look into it.